docs/components/Contribute.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <div>
+    <h1>You are welcome to contribute to our project! 🤝</h1>
+    <h2>📝 How to contribute to {{ name }}?</h2>
+    <p>
+      Our repository you can find by checking our <a href="https://github.com/EternalCodeTeam">GitHub</a> and selecting
+      the project
+      you want to contribute to.
+      Or you can access this repository directly, by clicking on the project name here:
+      <a :href='`https://github.com/EternalCodeTeam/${name}`' target="_blank">{{ name }}</a>
+    </p>
+    <h3> Step 1: Fork the repository 🍴 </h3>
+    <p>
+      Fork the repository by clicking the "Fork" button on the top, right corner of the page. This will create a copy of
+      the repository in your account.
+    </p>
+    <h3> Step 2: Clone the repository 🛎️ </h3>
+    <p>
+      Now clone the forked repository to your machine. Go to your GitHub account, open the forked repository, click on
+      the
+      code button and then click the copy to clipboard icon.
+    </p>
+    <p>
+      Open a terminal and run the following git command:
+    </p>
+    <div class="language-bash">
+      <code>
+        git clone "url you just copied"
+      </code>
+    </div>
+    <h3>Step 3: Create a branch 🌿</h3>
+    <p>
+      Change to the repository directory on your computer (if you are not already there):
+    </p>
+    <div class="language-bash">
+      <code>
+        cd {{ name }}
+
+      </code>
+    </div>
+    <p>
+      Now create a branch using the `git checkout` command:
+    </p>
+    <div class="language-bash">
+      <code>
+        git checkout -b
+        your-new-branch-name
+      </code>
+    </div>
```

Comment:
Nie wystarczy zrobić czegoś takiego -> `coś tam komand`, jak jest rozproszone na cały poziom konteneru na docsy to dość dziwnie to wygląda